src/wmtk/Mesh.cpp

```diff
@@ -113,6 +113,11 @@ bool Mesh::simplex_is_equal(const Simplex& s0, const Simplex& s1) const
     return (s0.primitive_type() == s1.primitive_type()) && (id(s0) == id(s1));
 }
 
+bool Mesh::simplex_is_equal(const simplex::Simplex& s0, const simplex::Simplex& s1) const
```

Comment:
perhaps `are_equal` (you have two items, so use are). I'm in general not a huge fan of the simplex class leaking into Mesh (perhaps a light utility class like `SimplexComparison{ const Mesh& m_mesh; };` would be good)?

Reply:
i think a new SimplicialComplex class should mediate between Mesh internals and Simplicial things - and SimplexComparison could be a friend of that?